docker-for-windows/index.md

```diff
@@ -307,75 +194,58 @@ access the mounted volumes.
-configurable. See the troubleshooting topic [Permissions errors on data
-directories for shared
-volumes](troubleshoot.md#permissions-errors-on-data-directories-for-shared-volumes) for workarounds and more detail.
+ * Docker for Windows sets permissions to read/write/execute for users and
+   read/execute for groups and others [0755 or u+rwx,go+rx](http://permissions-calculator.org/decode/0755/).
+   This is not configurable. See [Permissions errors on data directories for shared volumes](troubleshoot.md#permissions-errors-on-data-directories-for-shared-volumes).
 >
- * Make sure that the domain user has permissions to shared drives,
- as described in the troubleshooting topic ([Verify domain user has permissions for shared drives](troubleshoot.md#verify-domain-user-has-permissions-for-shared-drives-volumes)).
+ * Ensure the domain user has access to shared drives, as described in [Verify domain user has permissions for shared drives](troubleshoot.md#verify-domain-user-has-permissions-for-shared-drives-volumes).
 >
  * You can share local drives with your _containers_ but not with Docker Machine
-nodes. See [Can I share local drives and filesystem with my Docker Machine
-VMs?](faqs.md#can-i-share-local-drives-and-filesystem-with-my-docker-machine-vms) in the FAQs.
+nodes. See the FAQ, [Can I share local drives and filesystem with my Docker Machine VMs?](faqs.md#can-i-share-local-drives-and-filesystem-with-my-docker-machine-vms).
 >
 
 #### Firewall rules for shared drives
 
 Shared drives require port 445 to be open between the host machine and the virtual
 machine that runs Linux containers.
 
-> **Note**: Docker detects if port 445 is closed and shows
-the following message when you try to add a shared drive: ![Port 445
-blocked](images/drive_sharing_firewall_blocked.png)
+> Docker detects if port 445 is closed and shows the following message when you
+> try to add a shared drive: ![Port 445 blocked](images/drive_sharing_firewall_blocked.png)
 
 To share the drive, allow connections between the Windows host machine and the
 virtual machine in Windows Firewall or your third party firewall software. You
 do not need to open port 445 on any other network. By default, allow connections
-to 10.0.75.1 port 445 (the Windows host) from 10.0.75.2 (the virtual machine).
-If the firewall rules appear to be open, consider [reinstalling the File and
-Print Sharing service on the virtual network
-adapter.](http://stackoverflow.com/questions/42203488/settings-to-windows-firewall-to-allow-docker-for-windows-to-share-drive/43904051#43904051)
+to `10.0.75.1` port 445 (the Windows host) from `10.0.75.2` (the virtual machine).
+If the firewall rules appear to be open, consider [reinstalling the File and Print Sharing service on the virtual network adapter](http://stackoverflow.com/questions/42203488/settings-to-windows-firewall-to-allow-docker-for-windows-to-share-drive/43904051#43904051).
```

Comment:
You don't need to keep the title, you can just stick to `https://stackoverflow.com/questions/43904051`.

Reply:
Or `https://stackoverflow.com/a/43904051/1353549`, which is what you get if you click `Share` on that answer.